hydradx/notebooks/Omnipool/LP_returns_analysis3.ipynb

```diff
@@ -47,7 +44,8 @@
     "    'USD': {'usd price': 1, 'weight': 0.20},\n",
     "    'BTC': {'usd price': price_list[0]['BTC'], 'weight': 0.10},\n",
     "    'ETH': {'usd price': price_list[0]['ETH'], 'weight': 0.40},\n",
-    "    'INTR': {'usd price': price_list[0]['DOT'], 'weight': 0.20}\n",
+    "    'INTR': {'usd price': price_list[0]['DOT'], 'weight': 0.03},\n",
+    "    'TKN': {'usd price': price_list[0]['DOT'], 'weight': 0.17}\n",
```

Comment:
This looks incorrect, don't we want TKN to have 3% weight and DOT to have 17% weight, and we want to remove INTR altogether?

Reply:
You're right, that was incorrect.